docs/fxa-engineering/release-process.md

```diff
@@ -61,6 +61,18 @@ sidebar_label: Release Process
 
 1. Initial deployment bug is closed
 
+### Special Cases
+
+#### Releasing Icons 
+
+All product icons live in a dedicated directory in the fxa repo - `assets/product-icons`.
```

Comment:
Product icons for Stripe specifically live here, which was the reason for creating this process. But, we're uploading the whole `assets` directory to the CDN - so things can live in the directory above `product-icons` if we find other things that need handling like this

Reply:
Might also be worth mentioning that the CDN is hosted here: https://accounts-static.cdn.mozilla.net/

Reply:
Great, thanks for the extra info. Do you think it'd make sense to put the basic CDN-related facts (including the URL, the existence of the release script, and the mapping of the top-level `assets/` folder) in the FxA readme, and just put release-related stuff here? Seems like you might not look in the 'release' doc for basic CDN info.